sdk/storage/azure-storage-file-datalake/swagger/README.md

```diff
@@ -16,7 +16,7 @@ autorest --v3 --python
 
 ### Settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json
+input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/8d43652798dced3da898e53ed6e32b1d706a94d6/specification/storage/data-plane/Azure.Storage.Files.DataLake/preview/2021-06-08/DataLakeStorage.json
```

Comment:
Let's revert this before checking in. Could be done later before we merge feature branch but mine as well do it now.